### PR TITLE
Added CTA to landscape#smi

### LINKED ISSUE #1531: [SMI] CTA: for user to view landscape#smi at the bottom test details table
@dhruv0000 no, a Call To Action, a CTA - section and button at the bottom of the detailed tests table telling the viewer to see the current status of the support for SMI of all service meshes.

_Originally posted by @leecalcote in https://github.com/layer5io/layer5/issues/1526#issuecomment-778249558_

### PR DESCRIPTION
**Pull request by dhruv0000**
Signed-off-by: dhruv0000 <patel.4@iitj.ac.in>

**Description**

This PR fixes #1531

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
Looks good, just one thing instead of adding an `a` tag in the landscape page for the SMI table, can you instead add an `id` attribute to the table `div` or the `heading`. It works the same way, but cleaner is what I think.

File: src/sections/Projects/SMI/smi.style.js [hunk 1]
```diff
@@ -174,6 +174,32 @@ const SMIWrapper = styled.section`
 		}
 	}
 
+    .smiResults {
+        display: flex;
+		padding: 2.5rem;
+		text-align: center;
+		margin: auto;
+		margin-top: 4rem;
+		margin-bottom: 4rem;
```
Comment (Nikhil-Ladha) [inline src/sections/Projects/SMI/smi.style.js:183]: This is not needed. Adding too much white space.

Comment (Nikhil-Ladha) [inline src/sections/Projects/SMI/smi.style.js:183]: Shorthand form preferred.

File: src/sections/Projects/SMI/smi.style.js [hunk 2]
```diff
@@ -174,6 +174,32 @@ const SMIWrapper = styled.section`
 		}
 	}
 
+    .smiResults {
+        display: flex;
+		padding: 2.5rem;
```
Comment (Nikhil-Ladha) [inline src/sections/Projects/SMI/smi.style.js:179]: The bottom padding can be reduced. Too much white space.

**Comment by leecalcote** [comment]
@dhruv0000 or @Nikhil-Ladha, will you kindly reconcile the conflict in style. Thank you, gents.

**Comment by leecalcote** [comment]
Great work, gents. One final item: the "Check It Out" button opens to a new window (_blank)

**Comment by leecalcote** [comment]
@Tanuj22 what behavior should we expect out of `external`?

```
              <Button primary url="/service-mesh-landscape#smi" external="true">
```

**Comment by Nikhil-Ladha** [comment]
> @Tanuj22 what behavior should we expect out of `external`?
> 
> ```
>               <Button primary url="/service-mesh-landscape#smi" external="true">
> ```

`external={false}`

**Review by leecalcote — APPROVED** [review]

**Pull request merged**